static/app/views/settings/organizationGeneralSettings/organizationSettingsForm.spec.tsx

```diff
@@ -169,4 +171,63 @@ describe('OrganizationSettingsForm', function () {
       })
     );
   });
+
+  it('can toggle hideAiFeatures setting', async function () {
+    putMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/`,
+      method: 'PUT',
+      body: {...organization, hideAiFeatures: true},
+    });
+
+    render(
+      <OrganizationSettingsForm
+        {...routerProps}
+        initialData={OrganizationFixture({hideAiFeatures: false})}
+        onSave={onSave}
+      />,
+      {
+        organization: {
+          ...organization,
+          features: ['autofix'],
+        },
+      }
+    );
+
+    await userEvent.click(screen.getByRole('checkbox', {name: 'Hide AI Features'}));
+
+    expect(putMock).toHaveBeenCalledWith(
+      '/organizations/org-slug/',
+      expect.objectContaining({
+        data: {
+          hideAiFeatures: true,
+        },
+      })
+    );
+  });
+
+  it('disables hideAiFeatures toggle and shows tooltip for DE region', function () {
+    // Mock the region util to return DE region
+    jest.spyOn(RegionUtils, 'getRegionDataFromOrganization').mockImplementation(() => ({
+      name: 'de',
+      displayName: 'Europe (Frankfurt)',
+      url: 'https://sentry.de.example.com',
+    }));
```

Comment:
jest.mocked here preserves the return types of getRegionDataFromOrganization and you've already mocked the module.

```suggestion
    jest.mocked(RegionUtils.getRegionDataFromOrganization).mockImplementation(() => ({
      name: 'de',
      displayName: 'Europe (Frankfurt)',
      url: 'https://sentry.de.example.com',
    }));
```